_includes/tutorials/kafka-producer-application/kafka/markup/dev/make-application.adoc

```diff
@@ -0,0 +1,70 @@
+////
+In this file you describe the Kafka streams topology, and should cover the main points of the tutorial.
+The text assumes a method buildTopology exists and constructs the Kafka Streams application.  Feel free to modify the text below to suit your needs.
+////
+Before you create your application file, let's look at some of the key points of this program:
+
+[source, java]
+.KafkaProducerApplication constructor
+----
+
+public class KafkaProducerApplication {
+
+  private final Producer<String, String> producer;
+  final String outTopic;
+
+  public KafkaProducerApplication(final Producer<String, String> producer,  <1>
+                                  final String topic) {                     <2>
+    this.producer = producer;
+    outTopic = topic;
+  }
+
+----
+
+<1> Passing in the `Producer` instance as a constructor parameter.
+<2> The topic to write records to
+
+
+In this tutorial you'll inject the dependencies in the `KafkaProducerApplication.main()` method, but in practice you may want to use a dependency injection framework library, such as the  https://spring.io/projects/spring-framework[Spring Framework].
```

Comment:
What's the value of injecting Spring into this tutorial?  SEO? ;)

Reply:
I didn't think of it that way.  I added it as I think it reflects what goes on in practice.  I have a similar statement in the consumer tutorial. Do you think we should remove it?

Reply:
@bbejeck no strong leaning one way or the other, feel free to leave it in if it reflects what happens in practice.